main.go

```diff
@@ -16,6 +16,7 @@ func main() {
 	wtsURL := flag.String("wtsURL", "", "workspace-token-service url")
 	wtsIDP := flag.String("wtsIDP", "", "workspace-token-service IDP to use (optional)")
 	apiKey := flag.String("api-key", "", "api key")
+	accessToken := flag.String("access-token", "", "access token (optional)")
```

Comment:
-> Can you add info to the README that describes what this parameter does, what use case it matches, and how to populate it?

-> I think that the name "access token" is generic enough that it could mean anything in this context. Can you rename it to be more specific? For example, "ExternalClusterAccessToken" + "external-cluster-access-token"